src/TorchSharp/Utils/tensorboard/protobuf/AllocationDescription.cs

```diff
@@ -0,0 +1,337 @@
+// <auto-generated>
+//     Generated by the protocol buffer compiler.  DO NOT EDIT!
+//     source: tensorboard/compat/proto/allocation_description.proto
+// </auto-generated>
+#pragma warning disable 1591, 0612, 3021
+#region Designer generated code
+
+using pb = global::Google.Protobuf;
+using pbc = global::Google.Protobuf.Collections;
+using pbr = global::Google.Protobuf.Reflection;
+using scg = global::System.Collections.Generic;
+namespace Tensorboard {
+
+  /// <summary>Holder for reflection information generated from tensorboard/compat/proto/allocation_description.proto</summary>
+  public static partial class AllocationDescriptionReflection {
+
+    #region Descriptor
+    /// <summary>File descriptor for tensorboard/compat/proto/allocation_description.proto</summary>
+    public static pbr::FileDescriptor Descriptor {
+      get { return descriptor; }
+    }
+    private static pbr::FileDescriptor descriptor;
+
+    static AllocationDescriptionReflection() {
+      byte[] descriptorData = global::System.Convert.FromBase64String(
+          string.Concat(
```

Comment:
>string.Concat( [](http://example.com/codeflow?start=10&length=14)

nit: instead of calling it, you may just use `+` with the literal string?

Reply:
Those are files generated by the protoc compiler.

Reply:
Sorry, should have mentioned -- skip all the files in the 'protobuf' folder.

Reply:
That make the review faster :-) thanks!